rules/ast/functions.go

```diff
@@ -291,6 +291,25 @@ func dropCommonLabelsImpl(timestamp clientmodel.Timestamp, args []Node) interfac
 	return vector
 }
 
+// === round(vector VectorNode) Vector ===
+func roundImpl(timestamp clientmodel.Timestamp, args []Node) interface{} {
+	// round to even
```

Comment:
nit: we do all our comments sentence-style, that is capital first letter and period.